scale-ui/app/modules/jobs/partials/jobDetailTemplate.html

```diff
@@ -27,11 +27,20 @@ <h4 class="panel-title error-title"><strong>{{ vm.job.error.category }} Error: {
                         {{ vm.job.error.description }}
                     </div>
                 </div>
+                <div class="panel panel-info" ng-show="vm.job.is_superseded">
+                    <div class="panel-heading">
+                        <div class="panel-title">
+                            <strong>SUPERSEDED {{ vm.supersededChange }}</strong>
+                        </div>
+                    </div>
+                </div>
```

Comment:
Yeah, that's moving in the right direction. I would suggest dropping the colon and using a bit more space, maybe event right align the timestamp so the link stands out better.